vcsclient/azurerepos_test.go

```diff
@@ -98,6 +98,24 @@ func TestAzureRepos_TestDownloadRepository(t *testing.T) {
 	defer cleanUp()
 	err = client.DownloadRepository(ctx, "", repo1, branch1, dir)
 	require.NoError(t, err)
+	rootFiles, err := os.ReadDir(dir)
+	assert.NotEmpty(t, rootFiles)
+	assert.NoError(t, err)
+	readmeFound := false
+	dotGitFound := false
+	for _, file := range rootFiles {
+		if readmeFound && dotGitFound {
+			break
+		}
+		if file.Name() == "README.md" {
+			readmeFound = true
+		}
+		if file.Name() == ".git" {
+			dotGitFound = true
+		}
+	}
+	assert.True(t, readmeFound)
+	assert.True(t, dotGitFound)
```

Comment:
Please consider this:
```suggestion
	assert.FileExists(t, filepath.Join(dir, "README.md"))
	assert.DirExists(t, filepath.Join(dir, ".git"))
```

Reply:
done